js/app.js

```diff
@@ -1 +1,316 @@
-console.log('Hello frontend');
+document.addEventListener('DOMContentLoaded', function() {
+
+// Variables
+
+var boardArray = ["","","","","","","","",""];
+
+var count = 0;
+
+var win = 0;
+
+var spaces = document.querySelectorAll('div');
```

Comment:
Better to select by class "space". This is because you'll almost always have other divs in your page.